api/src/main/java/org/apache/cloudstack/api/response/IPAddressResponse.java

```diff
@@ -150,6 +150,10 @@
     @Param(description = "is public ip for display to the regular user", since = "4.4", authorized = {RoleType.Admin})
     private Boolean forDisplay;
 
+    @SerializedName("networkname")
```

Comment:
@ravening can you change that to use `ApiConstants`?

Reply:
@rhtyd Done